src/components/AuthScreen/AuthScreen.component.js

```diff
@@ -17,7 +17,7 @@ class AuthScreen extends Component {
         <WelcomeScreen
           heading={intl.formatMessage(messages.heading)}
           text={intl.formatMessage(messages.text)}
-          onClose={history.goBack}
+          onClose={() => history.replace('/board/root')}
```

Comment:
We should add a conditional check:

If the history stack contains a previous URL, use goBack to return the user to that URL (for example, when the login/signup flow is opened from the settings page).

If there is no previous URL in the history stack, redirect the user to the home page ('/').
Please also make sure the navigation is properly performed and that the URL actually changes as expected.

Reply:
@RodriSanchez1 Thank you for the feedback! This was my initial thought too in terms of user experience, but I understood the issue description as "_always_ go to the root board when the AuthScreen is closed":
> I found a bug related to the close action at the login screen.
> Supposed to return to the root board, not turn to the previous screen

The `history` object gives us a length but no visibility into the previous URL. Given that, we don't know _what_ the user's previous page was. It could have been the Settings page, but it could also have been google.com or even just the browser's New Tab screen. I want to confirm if it's OK to send the user back to the previous page as long as `history.length > 1`, even if we don't know what URL is at the top of the stack.

Reply:
@RodriSanchez1 committed! I also created a new issue here: #1949